ngrinder-frontend/src/js/components/perftest/report/menu/PerfTest.vue

```diff
@@ -130,7 +145,8 @@
             &.bigchart, &.chart {
                 border: 1px solid #c4c4c4;
                 height: 300px;
-                min-width: 615px;
+                min-width: 960px;
+                max-width: 960px;
```

Comment:
I think it would be as simple as 960 in width.
Why did you declare min-width and max-width separately?

Reply:
Fix that.